Content.Server/Abilities/Boxer/Boxer/BoxerComponent.cs

```diff
@@ -0,0 +1,32 @@
+using Content.Shared.Damage;
+
+namespace Content.Server.Abilities.Boxer
+{
+    /// <summary>
+    /// Added to the boxer on spawn.
+    /// </summary>
+    [RegisterComponent]
+    public sealed class BoxerComponent : Component
+    {
+        [DataField("paralyzeChanceNoSlowdown")]
+        public float ParalyzeChanceNoSlowdown { get; set; } = 0.2f;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("paralyzeChanceWithSlowdown")]
+        public float ParalyzeChanceWithSlowdown { get; set; } = 0.5f;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("paralyzeTime")]
+        public float ParalyzeTime { get; set; } = 5f;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("slowdownTime")]
+        public float SlowdownTime { get; set; } = 3f;
+
+        [DataField("modifiers", required: true)]
+        public DamageModifierSet UnarmedModifiers = default!;
+
+        [DataField("rangeBonus")]
+        public float RangeBonus = 1.5f;
+    }
```

Comment:
Looks like a copy of stunbaton? Also this should be on the gloves and not the job.

Reply:
I don't think we should be adding traits for every single flavor job. ATM it's just clowns being clumsy I think and mimes but at least mimes can stop being a mime.

Reply:
Forgot chaplain